### PR TITLE
Added minimum password length to app.ini

### PR DESCRIPTION
**Pull request by Bwko**
Adds feature #220

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

**Comment by strk** [comment]
LGTM (untested) - this goes to 1.1.0, right ?

**Comment by strk** [comment]
@lunny being a new feature, shouldn't this go to 1.1.0 ?

**Comment by Bwko** [comment]
I've updated the en-US translation to contain the %d that gets replaced by the minimum password length

**Comment by bkcsoft** [comment]
@strk @lunny the related issue is set to 1.1.0 so I changed the milestone here :)

**Comment by thibaultmeyer** [comment]
LGTM

**Comment by lunny** [comment]
Please resolve the conflict and we will merge it

**Comment by Bwko** [comment]
Resolved

File: modules/setting/setting.go [hunk 1]
```diff
@@ -589,6 +590,11 @@ please consider changing to GITEA_CUSTOM`)
 	CookieUserName = sec.Key("COOKIE_USERNAME").MustString("gitea_awesome")
 	CookieRememberName = sec.Key("COOKIE_REMEMBER_NAME").MustString("gitea_incredible")
 	ReverseProxyAuthUser = sec.Key("REVERSE_PROXY_AUTHENTICATION_USER").MustString("X-WEBAUTH-USER")
+	MinPasswordLength = sec.Key("MIN_PASSWORD_LENGTH").MustInt()
```
Comment (tboerger) [inline modules/setting/setting.go:593]: Just add 6 to mustint and you can drop the following.

Comment (lunny) [reply modules/setting/setting.go:593]: Really

**Pull request merged**